Remotion/Data/DomainObjects/Queries/Configuration/QueryDefinition.cs

```diff
@@ -109,6 +80,10 @@ public QueryDefinition (string queryID, StorageProviderDefinition storageProvide
   /// <param name="collectionType">If <paramref name="queryType"/> specifies a collection to be returned, <paramref name="collectionType"/> specifies the type of the collection.
   /// If <paramref name="queryType"/> is <see langword="null"/>, <see cref="DomainObjectCollection"/> is used.
   /// </param>
+  /// <param name="metaData">
+  /// Adds metadata to a query which should be used to communicate additional information for the query, like whether it should connect to the read/write or readonly instance of
+  /// the db.
```

Comment:
This comment suggests that there are two instances of the database (db), but it is only the connection that is designated as read-only, the database is always the same. Therefore I suggest the following:
"... like whether the query should be executed using a read-only connection."

Reply:
Huh, seems like I misunderstood how this is supposed to be used. Thanks